style.css

```diff
@@ -1 +1,245 @@
 /*  Add your styling here */
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300;400&display=swap');
```

Comment:
This works too. But nowadays, it is best to try and use <link> on the html for fonts rather than @import on the css. As it is better for resource load speeds, hence better for SEO.
**Improvement 6:** convert this font import to <link> on the index.html